### ISSUE TITLE
beginning damage depth

### ISSUE THREAD

**Issue by rnugent3** [HydrologicEngineeringCenter/HEC-FDA#595]
We need to handle beginning damage depth here: 

https://github.com/HydrologicEngineeringCenter/HEC-FDA/blob/62df772cf4c0c4d666fe6abf49b9196eec5bdb46/fda-model/structures/DeterministicStructure.cs#L41

**Comment by rnugent3** [comment]
I am going to set beginning damage depth to 0 feet by default. this way we don't check for beginning damage depth, we'll just always use it and it won't matter unless beginning damage depth has been identified by the user to be something different.

**Comment by rtle76** [comment]
So I should forget about this?

**Comment by rnugent3** [comment]
No, sir. We still need to handle beginning damage depth at the line indicated above. 

We had previously discussed checking to see if beginning damage depth is not null, and if not, truncating the stage-damage function. Now that I think this through, the default value should be -999. You can truncate the depth-percent damage function at the beginning damage depth for all structures - independent of whether the structure is actually using the parameter. This way, if the structures beginning damage depth is the default value of -999, it will be impossible to truncate the depth-percent damage function. Truncation should only occur for values of about -8 and above.

**Comment by rnugent3** [comment]
Part of this issue should be to reset the default value back to -999

**Comment by rtle76** [comment]
I'm tracking, thank you for clearing that up
